README.md

```diff
@@ -1,165 +1,17 @@
-| Title |Payment Title displayed during checkout |
-
-**Installment Payment Solution**  
-Available options:
-
-| Setting             | Description  |
-| ------------ | --------------------------------- |                                                                            
-| Enable/Disable | Enables or disables 'Installment Payment method'|
-| Title |Payment Title displayed during checkout |
-
-**True Money Payment Solution**  
-Available options:
-
-| Setting             | Description  |
-| ------------ | --------------------------------- |                                                                            
-| Enable/Disable | Enables or disables 'True Money Payment method'|
-| Title |Payment Title displayed during checkout |
-
-**Citi Pay With Points Payment Solution**  
-Available options:
-
-| Setting             | Description  |
-| ------------ | --------------------------------- |                                                                            
-| Enable/Disable | Enables or disables 'True Money Payment method'|
-| Title |Payment Title displayed during checkout |
-
-**PayNow Payment Solution (Available only for Singaporean customers)**  
-Available options:
-
-| Setting             | Description  |
-| ------------ | --------------------------------- |                                                                            
-| Enable/Disable | Enables or disables 'PayNow Payment method'|
-| Title |Payment Title displayed during checkout |
-
-**Convenience Store Payment Solution (Available only for Japanese customers)**  
-Available options:
-
-| Setting             | Description  |
-| ------------ | --------------------------------- |                                                                            
-| Enable/Disable | Enables or disables 'Convenience Store Payment method'|
-| Title |Payment Title displayed during checkout |
+See our [official docs](https://www.omise.co/magento-plugin) for details on installation and configuration.
 
 ## Contributing
 
-Thanks for your interest in contributing to Omise Magento. We're looking forward to hearing your thoughts and willing to review your changes.
-
-The following subjects are instructions for contributors who consider to submit changes and/or issues.
-
-### Submit the changes
-
-You're all welcome to submit a pull request.
-Please consider the [pull request template](https://github.com/omise/omise-magento/blob/master/.github/PULL_REQUEST_TEMPLATE.md) and fill the form when you submit a new pull request.
-
-Learn more about submitting pull request here: [https://help.github.com/articles/about-pull-requests](https://help.github.com/articles/about-pull-requests)
-
-### Submit the issue
-
-Submit the issue through [GitHub's issue channel](https://github.com/omise/omise-magento/issues).
-
-Learn more about submitting an issue here: [https://guides.github.com/features/issues](https://guides.github.com/features/issues)
-
-## License
```

Comment:
Removed license as there's a big button that says the license at the top of the repo.

![Screen Shot 2020-06-18 at 8 14 20 AM](https://user-images.githubusercontent.com/772616/84966439-c258ae80-b13b-11ea-8393-dc01cc4159ec.png)